requirements.in

```diff
@@ -66,5 +66,11 @@ django==4.2.15
 
 # NOTE(willkg): Need to keep elasticsearch and elasticsearch-dsl at these versions
 # because they work with the cluster we're using
-elasticsearch==1.9.0
-elasticsearch-dsl==0.0.11
+# NOTE(relud): these dependencies are installed separately, relocated, and patched to reference the
+# new location (aka shaded), so that they can be installed at the same time as newer versions
+# elasticsearch==1.9.0
+# elasticsearch-dsl==0.0.11
+# NOTE(relud): specify transitive dependencies that are not shaded
+python-dateutil # via elasticsearch-dsl
+six # via elasticsearch-dsl
+urllib3>=1.8, <2.0 # via elasticsearch
```

Comment:
Do you know that these versions are compatible with the old and new versions of the ES packages? Or does that come at a later step?

What happens if there isn't a single version of each transitive dependency that's version compatible with both the old and new versions of the ES packages?

Reply:
> Do you know that these versions are compatible with the old and new versions of the ES packages? Or does that come at a later step?

later step. this is just the versions as specified by the old versions of the packages.

> What happens if there isn't a single version of each transitive dependency that's version compatible with both the old and new versions of the ES packages?

then we have to shade that dependency too